janitor/functions/conditional_join.py

```diff
@@ -417,13 +414,12 @@ def _less_than_indices(
     where `left_c` is less than
     (but not equal to) `right_c` are returned.
 
-    A tuple of integer indexes for left_c and right_c is returned.
+    if multiple_conditions is False, a tuple of integer indexes
```

Comment:
I think we don't have `multiple_conditions` as an argument here, is that right, @samukweku? If so, does this part of the docstring need updating?

Reply:
I updated it.